lib/puppet_x/puppetlabs/aws.rb

```diff
@@ -162,6 +162,14 @@ def elb_client(region = default_region)
         self.class.elb_client(region)
       end
 
+      def self.elbv2_client(region = default_region)
```

Comment:
elbv2 is the interface to AWS's new application load balancer setup which supports path based routing at the application level. See http://docs.aws.amazon.com/cli/latest/reference/elbv2/index.html